source/community/monthly-community-meeting/index.md

```diff
@@ -24,6 +24,12 @@ layout: page
   </thead>
   <tbody>
   <tr>
+    <td>September 7, 2016</td>
+    <td><a href="https://www.dropbox.com/s/fp3p58n9wmfgnvt/PatternFly%20Community%20Meeting%202016-09.pdf?dl=0">View Deck</a></td>
+    <td><a target="_blank" href="https://bluejeans.com/s/aNH4r/">View Recording</a></td>
+  </tr>
+  <tr>
+  <tr>
```

Comment:
Good catch. There appears to be two extra <tr> tags if you view the full source.